README-zh-CN.md

```diff
@@ -0,0 +1,106 @@
+# dbt-oceanbase
+
+[OceanBase Database](https://github.com/oceanbase/oceanbase) 是一个分布式关系型数据库。完全由蚂蚁集团自主研发。OceanBase 基于 [Paxos](https://lamport.azurewebsites.net/pubs/lamport-paxos.pdf) 协议以及分布式架构，实现了高可用和线性扩展，与此同时 OceanBase 还是一款 HTAP 数据库，可以运行联机分析型负载。OceanBase 数据库运行在常见的服务器集群上，不依赖特殊的硬件架构。
+
+`dbt-oceanbase`将 OceanBase 与 DBT 集成在一起，允许用户使用 OceanBase 创建 DBT 项目或构建数据仓库。目前，本项目仅提供 OceanBase MySQL 兼容模式的支持，暂不支持 OceanBase Oracle 兼容模式。
+
+## 安装
+
+本项目使用 PyPi 进行管理，因此您需要预先安装 [setuptools](https://pypi.org/project/setuptools/)。您可以使用以下 shell 安装 dbt-oceanbase：
+```shell
+pip3 install dbt-oceanbase
```

Comment:
should we offer install from source option, like https://docs.getdbt.com/docs/core/source-install

Reply:
okay, I will fix this in next pr